threshold: 0.5
classes_by_category:
    people: [person]
classes:
    - person
max_sizes:
    - [0.5, 0.5, 0.5]
model_file: yolov5s.pt

subscribers:
  
  queue_size: 1
  exact_time: false
  slop: 0.2

  image_rgb: /butia_vision/bvb/image_rgb

  #points: /butia_vision/bvb/points
  
  image_depth: /butia_vision/bvb/image_depth

  camera_info: /butia_vision/bvb/camera_info

publishers:

  people_detection:
    topic: /butia_vision/br/recognition2D
    queue_size: 1

servers:

  list_classes:
    service: /butia_vision/br/people_detection/list_classes

  start:
    service: /butia_vision/br/people_detection/start

  stop:
    service: /butia_vision/br/people_detection/stop
